docs/apps/components/rate.md

```diff
@@ -0,0 +1,168 @@
+5. Interactive feedback loops
+6. Gathering quantitative data
+7. Market research surveys
+8. Website usability testing
+9. A/B testing
+
+### Interactivity
+
+1. **Tooltips**: Informative pop-ups explaining the implications or meaning of each rating level, especially for complex criteria.
+2. **Remember past ratings**: Pre-fill the rating based on the user's previous rating for the same item, saving time and effort.
+3. **Contextual recommendations**: Based on the user's rating, suggest similar items or content they might enjoy.
+4. **Adaptive interfaces**: Adjust the rating scale or feedback options based on the user's previous interactions or browsing behavior.
+
+## API
+
+### Events
+
+| **Name**| **Description**|
+|---------|----------------|
+| `onChange (element)| The onChange event occurs when the value of a field is changed|
+
+### Methods
+
+| **Name**| **Description**|**Parameters**|
+|---------|----------------|--------------|
+|`this.addClassName`|Add style class to a form item|`(fields: String`&#124;`String[], className: String)`|
+|`this.removeClassName`|Remove style class from a form item|`(fields: String`&#124;`String[], className: String)`|
+|`this.getComponent`|Returns a component whose id has been passed as a parameter|`(component_ID: String) : Object`|
+|`this.getValues`|Gets the current values of all fields|`() : Object`|
+|`this.hide`|Hides the field|`(fields: String`&#124;`String[])`|
+|`this.show`|Displays the field|`(fields: String`&#124;`String[])`|
+|`this.disable`| Disable rate field from user interaction|`(fields: String`&#124;`String[])`|
+|`this.enable`| Enable rate field from user interaction|`(fields: String`&#124;`String[])`|
+|`this.setData`|Set the data in the field. The Value object should be of type { componentId: componentValue }|`(Value: Object)`|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of the rate component in response to user input.
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+    4. Before using **this.enable(fields)**, make sure the component is **disabled**. This can be done using **this.disable(fields)** or by **enabling** the **Disabled checkbox** in the Component Attribute panel.
+
+#### Steps to use the methods for the Page Builder components
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/rate1.png">
```

Comment:
This image does not render the cancel button correctly. It is not showing the cancel text. I understand this was a bug in our platform earlier but that has been fixed now. Please take a new screenshot showing the buttons correctly. Everything else in this picture looks okay.

Reply:
done